pkg/conn/conn.go

```diff
@@ -98,10 +99,11 @@ func NewConnPool(cap int, newConn func(ctx context.Context) (*grpc.ClientConn, e
 // Mgr manages connections to a TiDB cluster.
 type Mgr struct {
 	*pdutil.PdController
-	tlsConf  *tls.Config
-	dom      *domain.Domain
-	storage  tikv.Storage
-	grpcClis struct {
+	tlsConf   *tls.Config
+	dom       *domain.Domain
+	storage   kv.Storage
+	tikvStore tikv.Storage
```

Comment:
Can we keep just one storage?

Reply:
Sadly we do need two storages. In our plan, `kv.Storage` contains coprocessor related interfaces, and `tikv.Storage` contains client-go specific interfaces (such as `GetLockResolver`). BR needs both interfaces, so we have to use different storages here.